lib/server.js

```diff
@@ -46,7 +63,7 @@ module.exports = (directory, format, width, height, framerate, horizontalFlip, v
       '-hls_segment_filename',
       path.join(directory, '%s-%%d.m4s'),
       '-hls_segment_type',
-      'fmp4'
+      'fmp4',
```

Comment:
Don't use trailing commas. I should probably codify this stuff with `prettier`.

Reply:
my prettier default setup caused these, `.prettierrc` would help for sure.